subprojects/docs/src/docs/release/notes.md

```diff
@@ -50,6 +50,27 @@ Example:
 ADD RELEASE FEATURES BELOW
 vvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvv -->
 
+<a name="SSL"></a>
```

Comment:
The name here is wrong, but regardless I don't think this is necessary? It should be generated.
```suggestion
```